server/common/Util.java

```diff
@@ -184,9 +182,8 @@ private static void configureRootLogger(Unfiltered defaultLogger, LoggerContext
         SizeAndTimeBasedRollingPolicy<?> policy = new SizeAndTimeBasedRollingPolicy<>();
         policy.setContext(context);
         policy.setFileNamePattern(logPath + TYPEDB_LOG_FILE_ARCHIVE_SUFFIX);
-        FileSize fileSize = FileSize.valueOf(outputType.fileSizeCap());
-        long directorySize = fileSize.getSize() + FileSize.valueOf(outputType.archivesSizeCap()).getSize();
-        policy.setMaxFileSize(fileSize);
+        long directorySize = outputType.fileSizeCap() + outputType.archivesSizeCap();
+        policy.setMaxFileSize(new FileSize(outputType.fileSizeCap()));
```

Comment:
refactor to use our own bytes size computation at parse time, simplifies this code and removes reliance on Logback!